src/common/libsubprocess/test/stdio.c

```diff
@@ -1395,6 +1473,8 @@ int main (int argc, char *argv[])
     test_stream_start_stop_initial_stop (r);
     diag ("stream_start_stop_mid_stop");
     test_stream_start_stop_mid_stop (r);
+    diag ("overflow_output_buffer");
+    test_overflow_output_buffer (r);
```

Comment:
Suggestoin: rename this test to something like `test_long_lines()` or similar since the point is to demonstrate that the buffer does _not_ overflow when it gets a long line.